publish.sbt

```diff
@@ -37,11 +37,3 @@ pomExtra := (
     </developer>
   </developers>
 )
-
-publishTo <<= version { v: String =>
```

Comment:
Note to self: will need to add back in way to publish to Sonatype.

Reply:
Since we already have this snippet, I just removed that piece of code.
https://github.com/tim-group/sbt-utc/blob/5e034b78393ab47c7db17b2d1486fb88744354cf/publish.sbt#L11-L17
BTW, I did publish SNAPSHOT to Sonatype without that piece of code.